tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -411,9 +520,27 @@ def test_stack(xp, shapes, axis):
     ((6,), numpy.array([], numpy.int32), 0),
     ((6,), numpy.array([], numpy.float64), 0),
 ])
-def test_split(xp, shape, indices_or_sections, axis):
-    a = array_utils.create_dummy_ndarray(xp, shape, 'float32')
-    return xp.split(a, indices_or_sections, axis)
+class TestSplit(op_utils.NumpyOpTest):
+
+    def setup(self):
+        # TODO(niboshi): There's a bug in backward of split() in which the
+        # gradient shape differs from the input if indices are not in the
+        # sorted order. Fix this.
```

Comment:
Oh, should we create an issue or a card in the project backlog?

Reply:
Done: https://github.com/chainer/chainer/projects/2#card-19292911